server/src/test/scala/za/co/absa/atum/server/api/repository/PartitioningRepositorySpec.scala

```diff
@@ -91,10 +114,48 @@ class PartitioningRepositorySpec extends ZIOSpecDefault with TestData {
             failsWithA[DatabaseError]
           )
         }
+      ),
+
+      suite("GetPartitioningMeasuresSuite")(
+        test("Returns expected Seq") {
+          for {
+            result <- PartitioningRepository.getPartitioningMeasures(partitioningSubmitDTO1)
+          } yield assertTrue(result.getOrElse(Seq()).isEmpty)
+        },
+        test("Returns expected Exception") {
+          assertZIO(PartitioningRepository.getPartitioningMeasures(partitioningSubmitDTO2).exit)(
+            failsWithA[DatabaseError]
+          )
+        },
+        test("Returns expected Exception") {
+          assertZIO(PartitioningRepository.getPartitioningMeasures(partitioningSubmitDTO3).exit)(
+            failsWithA[DatabaseError]
+          )
+        }
+      ),
+
+      suite("GetPartitioningAdditionalDataSuite")(
+        test("Returns expected Right with empty Map") {
+          for {
+            result <- PartitioningRepository.getPartitioningAdditionalData(partitioningSubmitDTO1)
+          } yield assertTrue(result.isInstanceOf[AdditionalDataDTO])
```

Comment:
btw why you are testing only `isInstanceOf` and not an exact result?

Reply:
I just wanted to make sure that we are getting the expected instance.

Reply:
I understand that, but it was a rhetorical question. You can test exact & expected result instead